### PR TITLE
feat: add dynamic form for datasources for api's

### LINKED ISSUE #675: Make datasources populate from a json schema
Create a json schema for populating the datasources ui

### PR DESCRIPTION
**Pull request by gondar00**
This PR adds dynamic forms to datasources so that datasources can be populated using json schema

fixes #675 

Testing checklist

- [x] Postgres
- [x] Mysql
- [x] Sql server
- [x] Mongodb
- [x] Firestore
- [x] Dynamodb
- [x] Elasticsearch
- [x] Redis
- [x] Rest api
- [x] Graphql
- [x] Stripe
- [x] Airtable
- [x] Google sheets
- [x] Slack

### CODE REVIEW

File: frontend/src/Editor/DataSourceManager/SourceComponents/index.js [hunk 1]
```diff
@@ -9,10 +13,11 @@ import { Googlesheets } from './Googlesheets';
 import { Slack } from './Slack';
 import { Mongodb } from './Mongodb';
 import { Dynamodb } from './Dynamodb';
-import { Airtable } from './Airtable';
+// import { Airtable } from './Airtable';
```
Comment (Ckbhatia) [inline frontend/src/Editor/DataSourceManager/SourceComponents/index.js:16]: Comment the reason

File: frontend/src/Editor/DataSourceManager/SourceComponents/Database/Postgresql.schema.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    },
+    "ssl_enabled": {
+      "$label": "SSL",
+      "$key": "ssl_enabled",
+      "type": "toggle",
+      "description": "Toggle for ssl_enabled"
+    },
+    "database": {
+      "$label": "Database",
+      "$key": "port",
```
Comment (Navaneeth-pk) [inline frontend/src/Editor/DataSourceManager/SourceComponents/Database/Postgresql.schema.json:28]: shouldn't the key be `database`?

Comment (gondar00) [reply frontend/src/Editor/DataSourceManager/SourceComponents/Database/Postgresql.schema.json]: Yeah, fixed this part

File: frontend/src/Editor/DataSourceManager/SourceComponents/Database/Mssql.schema.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://json-schema.org/",
+  "$id": "https://tooljet.io/Mssql.schema.json",
+  "title": "Mongodb datasource",
```
Comment (akshaysasidrn) [inline frontend/src/Editor/DataSourceManager/SourceComponents/Database/Mssql.schema.json:4]: correction here

File: frontend/src/Editor/DataSourceManager/SourceComponents/Database/Redis.schema.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://json-schema.org/",
+  "$id": "https://tooljet.io/Dynamodb.schema.json",
```
Comment (akshaysasidrn) [inline frontend/src/Editor/DataSourceManager/SourceComponents/Database/Redis.schema.json:3]: correction here too

File: frontend/src/Editor/DataSourceManager/SourceComponents/Database/Mssql.schema.json (hunk 3 above)
Comment (gondar00) [reply frontend/src/Editor/DataSourceManager/SourceComponents/Database/Mssql.schema.json]: done 👍

File: frontend/src/Editor/DataSourceManager/SourceComponents/Database/Redis.schema.json (hunk 4 above)
Comment (gondar00) [reply frontend/src/Editor/DataSourceManager/SourceComponents/Database/Redis.schema.json]: done 👍

**Comment by akshaysasidrn** [comment]
looks good

**Pull request merged**